src/utils.c

```diff
@@ -2031,6 +2031,199 @@ gchar **utils_strv_join(gchar **first, gchar **second)
 	return strv;
 }
 
+/* * Returns the common prefix in a list of strings.
+ *
+ * The size of the list may be given explicitely automatically determined if passed a GStrv.
```

Comment:
I was under the impression that GStrv also implied NULL termination since it's expected by `g_strv_length` and `g_strv_contains`. I will change the comment.